scripts/dts/edtlib.py

```diff
@@ -1250,6 +1247,11 @@ def _check_binding(binding, binding_path):
             _err("missing, malformed, or empty 'description' for '{}' in "
                  "'properties' in {}".format(prop_name, binding_path))
 
+        enum = options.get("enum")
+        if enum and not isinstance(enum, list):
+            _err("enum in {} for property ({}) is not "
+                 "a list".format(binding_path, prop_name))
```

Comment:
Could do this to be consistent with the other checks:

```python
if "enum" in options and not isinstance(options["enum"], list):
    _err("enum in {} for property '{}' is not a list"
         .format(binding_path, prop_name))
```

Reply:
Changed.